src/Tribe/Metabox.php

```diff
@@ -345,6 +346,123 @@ public function ajax_ticket_delete() {
 		wp_send_json_success( $return );
 	}
 
+	/**
+	 * Sanitizes the data for the duplicate ticket ajax call, then duplicates the ticket and meta.
+	 *
+	 * @since TBD.
+	 */
+	public function ajax_ticket_duplicate() {
+		$post_id = absint( tribe_get_request_var( 'post_id', 0 ) );
+
+		if ( ! $post_id ) {
+			wp_send_json_error( esc_html__( 'Invalid parent Post', 'event-tickets' ) );
+		}
+
+		$ticket_id = absint( tribe_get_request_var( 'ticket_id', 0 ) );
+
+		if ( ! $ticket_id ) {
+			wp_send_json_error( esc_html( sprintf( __( 'Invalid %s', 'event-tickets' ), tribe_get_ticket_label_singular( 'ajax_ticket_duplicate_error' ) ) ) );
+		}
+
+		if ( ! $this->has_permission( $post_id, $_POST, 'duplicate_ticket_nonce' ) ) {
+			wp_send_json_error( esc_html( sprintf( __( 'Failed to duplicate the %s. Refresh the page to try again.', 'event-tickets' ), tribe_get_ticket_label_singular( 'ajax_ticket_duplicate_error' ) ) ) );
+		}
+
+		$provider = tribe_tickets_get_ticket_provider( $ticket_id );
+
+		if ( empty( $provider ) || ! $provider instanceof Tribe__Tickets__Tickets ) {
+			return new WP_Error(
+				'bad_request',
+				__( 'Commerce Module invalid', 'event-tickets' ),
+				[ 'status' => 400 ]
+			);
+		}
+		
+		// Get ticket data.
+		$ticket = $provider->get_ticket( $post_id, $ticket_id );
+		
+		if ( ! $ticket instanceof Tribe__Tickets__Ticket_Object ) {
+			return new WP_Error(
+				'bad_request',
+				__( 'Ticket ID invalid', 'event-tickets' ),
+				[ 'status' => 400 ]
+			);
+		}
+		
+		// Create data for duplicate ticket.
+		$data = [
+			'ticket_name'             => $ticket->name . __( ' (duplicate)', 'event-tickets'),
+			'ticket_description'      => $ticket->description,
+			'ticket_price'            => $ticket->price,
+			'ticket_show_description' => $ticket->show_description,
+			'ticket_start_date'       => $ticket->start_date,
+			'ticket_start_time'       => $ticket->start_time,
+			'ticket_end_date'         => $ticket->end_date,
+			'ticket_end_time'         => $ticket->end_time,
+			'tribe-ticket'            => [
+				'capacity' => $ticket->capacity(),
+				'mode'     => $ticket->global_stock_mode(),
+			]
+		];
+
+		// Add the ticket.
+		$duplicate_ticket_id = $provider->ticket_add( $post_id, $data );
+
+		// Successful?
+		if ( $duplicate_ticket_id ) {
+			
+			// Copy ticket meta from old ticket to new ticket.
+			$ignore_meta = [
+				'_sku',
+				'_tribe_ticket_manual_updated',
+				'_wp_old_slug',
+				'total_sales',
+			];
+			$ticket_meta = get_post_meta( $ticket->ID );
+			
+			if ( $ticket_meta ) {
+				foreach ( $ticket_meta as $meta_key => $meta_values ) {
+					
+					// Skip meta we don't want to duplicate.
+					if ( strpos( $meta_key, '_tec_tc_ticket_status_count' ) !== false ){
```

Comment:
```suggestion
					if ( false !== strpos( $meta_key, '_tec_tc_ticket_status_count' ) ) {
```
We need to use yoda conditional here.